Userland/Libraries/LibJS/Interpreter.cpp

```diff
@@ -140,6 +141,7 @@ void Interpreter::exit_scope(const ScopeNode& scope_node)
 
 void Interpreter::enter_node(const ASTNode& node)
 {
+    vm().call_frame().current_node = &node;
```

Comment:
This seems redundant now that `current_node` is set right after each `CallFrame call_frame;`

Reply:
We need to update the current call frame at each node to track execution progress within a frame, not just on each new call frame.